src/main/java/io/openliberty/tools/intellij/actions/LibertyProjectAction.java

```diff
@@ -129,6 +132,15 @@ protected final String[] buildFileToProjectNames(@NotNull List<BuildFile> list)
         return projectNames;
     }
 
+    protected final String[] buildFileToProjectPaths(@NotNull List<BuildFile> list) {
```

Comment:
Can we write a single method which returns array of both projectname and project path by using HashMap?

Reply:
Yes. Made the change. @aparnamichael